### ISSUE TITLE
Vide causes bandwidth leak/overages

### ISSUE THREAD

**Issue by dantheman213** [VodkaBears/Vide#204]
Vide doesn't do anything to prevent endless cycle of download and streaming to a small video file. I'm looping a 30mb file but this browser has downloaded over 5x that in just a few mins of having the page up in my browser.

![capture](https://cloud.githubusercontent.com/assets/10934533/22399667/ae96a028-e556-11e6-99e3-cfe58e10dba0.JPG)

**Comment by quinns** [comment]
I don't think it actually is. Make sure you have "Disable Cache (while DevTools is open)" checkbox disabled in the Network tab. This should cause your browser to not re-download the video for each loop, and better mimics real-world behavior.

![screen shot 2017-04-30 at 7 05 01 pm](https://cloud.githubusercontent.com/assets/150952/25570252/fedbe39e-2dd7-11e7-9cff-3751ed96c33d.png)
